### PR TITLE
Clientcfg bootstrap repo #12786 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

 Create Bootstrap Repos for EOL products
#12786

# Target branches

Which product version does this PR apply to?

- [x] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [ ] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12786

### CODE REVIEW

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 1]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 ====
 
 
+
+== Bootstrap Repository for End-of-Life Product
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:55]:
```suggestion
== Bootstrap Repository for End-of-Life Products
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 2]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
+
+Bootstrap repositories of supported products are automatically created and regenerated on the {productname} Server for every synchronized product.
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:57]:
```suggestion
When supported products are synchronized, bootstrap repositories are automatically created and regenerated on the {productname} Server.
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 3]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
+But if a product passes end-of-life and is unsupported, you must create a bootstrap repository for such a product manually in case you need it.
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:58]:
```suggestion
When a product reaches end-of-life and becomes unsupported, bootstrap repositories must be created manually if you want to continue using the product.
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 4]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
+
+For more information about bootstrap repositories, see xref:client-configuration:bootstrap-repository.adoc[].
+
+
+
+.Procedure: Creating Bootstrap Repositories for End-Of-Life Products
+
+. At the command prompt on the {productname} Server, as root, list the available, but unsupported bootstrap repositories with the [option]``--force`` option, for example:
+
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:67]:
```suggestion
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 5]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
+Bootstrap repositories of supported products are automatically created and regenerated on the {productname} Server for every synchronized product.
+But if a product passes end-of-life and is unsupported, you must create a bootstrap repository for such a product manually in case you need it.
+
+For more information about bootstrap repositories, see xref:client-configuration:bootstrap-repository.adoc[].
+
+
+
+.Procedure: Creating Bootstrap Repositories for End-Of-Life Products
+
+. At the command prompt on the {productname} Server, as root, list the available, but unsupported bootstrap repositories with the [option]``--force`` option, for example:
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:66]:
```suggestion
. At the command prompt on the {productname} Server, as root, list the available unsupported bootstrap repositories with the [option]``--force`` option, for example:
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 6]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
+
++
+----
+mgr-create-bootstrap-repo --list --force
+1. SLE-11-SP4-x86_64
+2. SLE-12-SP2-x86_64
+3. SLE-12-SP3-x86_64
+----
+
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:75]:
```suggestion
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 7]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
+. Create the bootstrap repository, using the appropriate repository name as the product label:
+
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:77]:
```suggestion
```

File: modules/client-configuration/pages/tshoot-clients.adoc [hunk 8]
```diff
@@ -51,6 +51,39 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
...
++
+----
+mgr-create-bootstrap-repo --create SLE-12-SP2-x86_64 --force
+----
+
```
Comment (Loquacity) [inline modules/client-configuration/pages/tshoot-clients.adoc:82]:
```suggestion
```

**Review by Loquacity — COMMENTED** [review]
Couple of wording changes, plus removed a whole lot of extra whitespace. Don't forget to edit what Vale picked up in these file, too.

**Comment by keichwa** [comment]
Thanks for all the improvements you posted!  Now waiting for linting results :)

**Review by Loquacity — APPROVED** [review]

**Pull request merged**